alex/applications/PublicTransportInfoEN/hclg/build_hclg.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
```

Comment:
You did not merged properly. The code in master changed for hclg and lm building. See 
https://github.com/UFAL-DSG/alex/tree/master/alex/applications/PublicTransportInfoEN/hclg

This script is for example in https://github.com/UFAL-DSG/alex/blob/master/alex/corpustools/kaldi_build_hclg.sh
and shared.